pkg/controller/storageprofile-controller_test.go

```diff
@@ -484,12 +487,60 @@ var _ = Describe("Storage profile controller reconcile loop", func() {
 		Expect(*sp.Status.StorageClass).To(Equal(storageClassName))
 		Expect(sp.Status.ClaimPropertySets).To(BeEmpty())
 
-		labels := createLabels(storageClassName, provisioner, false, true, false, false, false)
+		labels := createLabels(storageClassName, provisioner, isComplete, true, false, false, false, true)
 		Expect(int(metrics.GetStorageProfileStatus(labels))).To(Equal(count))
 	},
-		Entry("Noobaa (not supported)", storagecapabilities.ProvisionerNoobaa, 0),
-		Entry("Unknown provisioner", "unknown-provisioner", 1),
+		Entry("Noobaa (not supported)", storagecapabilities.ProvisionerNoobaa, false, 0),
+		Entry("Noobaa (not supported)", storagecapabilities.ProvisionerNoobaa, true, 1),
+		Entry("Unknown provisioner", "unknown-provisioner", false, 1),
+		Entry("Unknown provisioner", "unknown-provisioner", true, 0),
```

Comment:
Same entry name will be confusing when one of these fails

Reply:
Sure, these entries are not needed and can be merged. Fixed.